### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExplorer_ScreenReader: Screenreader does not read the necessary headings on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52010]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Explorer"(Cntrl+Shift+E) button.
3. Navigate to complementary landmark- "Explorer" section and read the content "Explorer" using basic navigation.

**Actual:**
Screenreader does not read the content "Explorer" as appopriate heading which is so.

**Same issue has been observed with JAWS.**

**Expected:**
Screenreader should read- "Explorer" as heading with appropriate heading level.

**Recommendations:**
Add appropriate native heading level tag to the content "Explorer".
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the appropriate headings present on the screen.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ViewExplorer_ScreenReader_Headings.pptx](https://github.com/Microsoft/vscode/files/2105626/A11y_VSCode_ViewExplorer_ScreenReader_Headings.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51760
